content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -0,0 +1,95 @@
+<!-- overview -->
+
+This document describes the concept of a VolumeAttributesClass in Kubernetes. Familiarity with [storage classes](https://kubernetes.io/docs/concepts/storage/storage-classes/), [volumes](https://kubernetes.io/docs/concepts/storage/volumes/) and [persistent volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) is suggested.
```

Comment:
Please save some extra text when appropriate. Also please use relative paths for links. Please wrap the long lines for ease of PR review, change tracking and
downstream localization.

```suggestion
This page assumes that you are familiar with [StorageClasses](/docs/concepts/storage/storage-classes/),
[volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
in Kubernetes.
```